spec/spec_helper.rb

```diff
@@ -1,85 +1,19 @@
 # This file is managed via modulesync
 # https://github.com/voxpupuli/modulesync
 # https://github.com/voxpupuli/modulesync_config
-RSpec.configure do |c|
-  c.mock_with :rspec
-end
-
-require 'puppetlabs_spec_helper/module_spec_helper'
-require 'rspec-puppet-facts'
-require 'bundler'
-include RspecPuppetFacts
-
-if ENV['DEBUG']
-  Puppet::Util::Log.level = :debug
-  Puppet::Util::Log.newdestination(:console)
-end
 
-# Rough conversion of grepping in the puppet source:
```

Comment:
These custom facts should be kept in. I started to work on support for this in voxpupuli-test but until then, please keep it.